rootcmd/root.go

```diff
@@ -96,6 +96,12 @@ func assignPersistentFlags(cmd *cobra.Command, flags *config.GlobalFlagValues) {
 	cmd.PersistentFlags().BoolVar(&flags.Verbose, "verbose", false, "verbose output")
 	cmd.PersistentFlags().BoolVar(&flags.NoColor, "no-color", false, "disable colors")
 	cmd.PersistentFlags().BoolVar(&flags.NoAutocompletion, "no-completion", false, "disable completion [interactive mode]")
+	cmd.PersistentFlags().StringVar(&flags.LogFile, "log-file", "", "direct logs to specified file")
+	cmd.PersistentFlags().StringVar(&flags.LogLevel, "log-level", "", fmt.Sprintf("(default 'error') write logs with the specified or higher level of importance. Log levels: %s", strings.Join(config.ValidLogLevels, ",")))
```

Comment:
```suggestion
	cmd.PersistentFlags().StringVar(&flags.LogLevel, "log-level", "", fmt.Sprintf("(default 'error') write logs with the specified or higher level of importance. Log levels: %s", strings.Join(config.ValidLogLevels, ", ")))
```

The message looks better this way IMO.